### PR TITLE
Fix 1.8 Blocking Position Accuracy

### PR DESCRIPTION
**Pull request by lowercasebtw**
Fixes slightly incorrect values in the blocking position code, to match 1.8 (edited values from the first 1.9 snapshot)

### CODE REVIEW

File: src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/item/MixinHeldItemRenderer.java [hunk 1]
```diff
@@ -51,16 +51,14 @@ private void transformLegacyBlockAnimations(AbstractClientPlayerEntity player, f
...
             applySwingOffset(matrices, arm, swingProgress);
-            matrices.translate(arm == Arm.RIGHT ? -0.14F : 0.14F, 0.12F, 0.12F);
-        } else {
-            matrices.translate(arm == Arm.RIGHT ? -0.15F : 0.15F, 0.07F, 0.12F);
         }
-        matrices.multiply(RotationAxis.POSITIVE_X.rotationDegrees(-102.25f));
-        matrices.multiply((arm == Arm.RIGHT ? RotationAxis.POSITIVE_Y : RotationAxis.NEGATIVE_Y).rotationDegrees(13.365f));
-        matrices.multiply((arm == Arm.RIGHT ? RotationAxis.POSITIVE_Z : RotationAxis.NEGATIVE_Z).rotationDegrees(78.05f));
+        
+        matrices.translate(-0.14142136F * (arm == Arm.RIGHT ? 1 : -1), 0.08F, 0.14142136F);
```
Comment (FlorianMichael) [inline src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/item/MixinHeldItemRenderer.java:58]: It is questionable if the 1.7 block hit animation still looks correctly, also the operation for x can be simplified.

Comment (lowercasebtw) [reply src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/item/MixinHeldItemRenderer.java:58]: The 1.7 item held values will be in the next pr

Comment (FlorianMichael) [reply src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/item/MixinHeldItemRenderer.java:58]: Your changes are breaking 1.7, they need to get pushed together.

Comment (lowercasebtw) [reply src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/item/MixinHeldItemRenderer.java:58]: ok gimme a second

Comment (FlorianMichael) [reply src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/item/MixinHeldItemRenderer.java]: Actually nevermind, this change did not broke 1.7 blocking. Your other fix can go into a separate pull request.

File: src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/item/MixinHeldItemRenderer.java [hunk 2]
```diff
@@ -51,16 +67,15 @@ private void transformLegacyBlockAnimations(AbstractClientPlayerEntity player, f
...
             applySwingOffset(matrices, arm, swingProgress);
-            matrices.translate(arm == Arm.RIGHT ? -0.14F : 0.14F, 0.12F, 0.12F);
-        } else {
-            matrices.translate(arm == Arm.RIGHT ? -0.15F : 0.15F, 0.07F, 0.12F);
         }
-        matrices.multiply(RotationAxis.POSITIVE_X.rotationDegrees(-102.25f));
-        matrices.multiply((arm == Arm.RIGHT ? RotationAxis.POSITIVE_Y : RotationAxis.NEGATIVE_Y).rotationDegrees(13.365f));
-        matrices.multiply((arm == Arm.RIGHT ? RotationAxis.POSITIVE_Z : RotationAxis.NEGATIVE_Z).rotationDegrees(78.05f));
+        
+        final int direction = arm == Arm.RIGHT ? 1 : -1;
```
Comment (FlorianMichael) [inline src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/item/MixinHeldItemRenderer.java:74]: Move this field directly below the Arm field and add a space between the direction int and the if (blockHitAnimation) part.

**Review by FlorianMichael — APPROVED** [review]
Looks good otherwise, it might be worth adding a comment to the code explaining where the values are from for later, also might consider renaming the function to "blockAnimation1_8"

**Comment by lowercasebtw** [comment]
Made requested changes

**Pull request merged**